packages/devtools_app/lib/src/primitives/custom_pointer_scroll_view.dart

```diff
@@ -332,22 +326,22 @@ class _CustomPointerScrollableState extends State<CustomPointerScrollable>
   /// To control what kind of [ScrollPosition] is created for a [Scrollable],
   /// provide it with custom [ScrollController] that creates the appropriate
   /// [ScrollPosition] in its [ScrollController.createScrollPosition] method.
-  ScrollPosition get position => _position;
-  ScrollPosition _position;
+  ScrollPosition? get position => _position;
```

Comment:
force unwrap _position and make this non nullable
https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/widgets/scrollable.dart#L391

Reply:
tests start failing if I do it
for example, this one: flutter test test/extent_delegate_list_view_test.dart